### PR TITLE
Optical flow simulator and install cleanup.

### PR DESCRIPTION
**Pull request by jgoppert**
This fixes errors with optical flow sim and has been tested in gazebo to work well with LPE estimator. It also adds install targets for mainapp in posix.

### CODE REVIEW

File: src/modules/local_position_estimator/BlockLocalPositionEstimator.cpp [hunk 1]
```diff
@@ -25,8 +25,8 @@ BlockLocalPositionEstimator::BlockLocalPositionEstimator() :
 	// subscriptions, set rate, add to list
 	_sub_armed(ORB_ID(actuator_armed), 1000 / 2, 0, &getSubscriptions()),
 	_sub_att(ORB_ID(vehicle_attitude), 1000 / 100, 0, &getSubscriptions()),
-	// flow 10 hz
-	_sub_flow(ORB_ID(optical_flow), 1000 / 10, 0, &getSubscriptions()),
+	// flow up to 50 hz, important we don't lose packets
```
Comment (julianoes) [inline src/modules/local_position_estimator/BlockLocalPositionEstimator.cpp]: Does this comment match the line below?

File: posix-configs/SITL/init/rcS_lpe_jmavsim_iris [hunk 2]
```diff
@@ -41,26 +50,31 @@ accelsim start
 barosim start
 adcsim start
 gpssim start
+#gps start -d /dev/ttyACM0 -s
```
Comment (julianoes) [inline posix-configs/SITL/init/rcS_lpe_jmavsim_iris]: Is this a debugging leftover?

Comment (jgoppert) [inline posix-configs/SITL/init/rcS_lpe_jmavsim_iris]: That was in rcS_jmavsim_iris, just tried to minimize the diff with lpe config.

**Comment by jgoppert** [comment]
Note, change of stil_gazebo to sim_update branch is just temporary to see if it builds on mac before I push to sitl_gazebo master.

**Pull request merged**